<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartLearningSystemGUI</class>
 <widget class="QDialog" name="SmartLearningSystemGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="result_label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>0</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>辨識結果:</string>
   </property>
  </widget>
  <widget class="QLabel" name="warning_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>640</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Warning:光線不足</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_btn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>640</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>新增單字至資料庫</string>
   </property>
  </widget>
  <widget class="QPushButton" name="revise_btn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>590</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>修正</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>741</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="camera_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>741</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ThisLabelWillBeChangedToVideoFrame</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="result_list">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>40</y>
     <width>351</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="revise_textbox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>590</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
